src/components/bcros/businessDetails/Links.vue

```diff
@@ -41,10 +42,50 @@ const isChangeBusinessInfoDisabled = computed(() => {
   return !isAllowed
 })
 
+const legalName = computed(() => currentBusiness.value?.legalName)
+// TO-DO: in the old codebase, the legalName is returned by the getLegalName() function.
+// Need to investigate the logic and implement it in business store.
```

Comment:
Can we create a ticket if it's not going to be done as part of  this ticket and then add new ticket number in the comment, thanks !